python/Ganga/Runtime/GPIexport.py

```diff
@@ -18,6 +18,31 @@
 from Ganga.GPIDev.Base.Objects import GangaObject
 from inspect import isclass
 
+def _addToInterface(interface, name, _object):
+
+    if isType(_object, GangaObject):
+        setattr(interface, name, addProxy(_object))
+    elif isclass(_object) and issubclass(_object, GangaObject):
+        setattr(interface, name, getProxyClass(_object))
+    else:
+        setattr(interface, name, _object)
+
+def exportToInterface(myInterface, name, _object, doc_section, docstring=None):
```

Comment:
There are many things 'in-front' of the proxy which aren't available from the Plugin system ```jobs``` is just one of these.
There are also situations where you don't a-prioiri know what you're looking for other than it's in front of the proxy.